MdeModulePkg/Library/VarCheckHiiLib/VarCheckHiiLibCommon.c

```diff
@@ -0,0 +1,353 @@
+/** @file
+  Var Check Hii Lib Common logic
+Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+#include <Uefi.h>
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>

```

Comment:
The following header files are not needed for this source file.

#include <Library/MemoryAllocationLib.h>
#include <Library/MmServicesTableLib.h>
#include <Protocol/MmCommunication.h>
#include <Library/VarCheckLib.h>